src/TLuaInterpreter.cpp

```diff
@@ -17756,6 +17756,10 @@ int TLuaInterpreter::setConfig(lua_State * L)
         host.mAnnounceIncomingText = getVerifiedBool(L, __func__, 2, "value");
         return success();
     }
+    if (key == qsl("translateNormalExitsInSpeedWalkDirTable")) {
```

Comment:
40 characters is far too long, can we halve it?

Reply:
:thinking: 
Would `translateSpeedWalkExits` be short enough - remember this is a term that will appear *just **once*** in an XML (so intended/designed to be both human and machine readable) - so clarity of meaning is perhaps more important than length - although conciseness is not to be completely ignored...